### PR TITLE
Issue and PR-Template

### LINKED ISSUE #427: Add new Issue-Templates
Adds new bug and feature templates.

### PR DESCRIPTION
**Pull request by steull**
closes #427

### CODE REVIEW

**Comment by FlorianK13** [comment]
@steull Thanks for these templates. Could you change the target branch to develop? Then we will merge it to production within the next Release.

**Comment by FlorianK13** [comment]
Just saw that I can also change the base branch

**Comment by jh-RLI** [comment]
Hey, I asked @steull  if he rolls out the templates in all open energy family repos. We are currently developing them internally at RLI. I think it will not change much more. So feel free to merge and use them already. This is also a small part of my master thesis, so feel free to give feedback on whether you are comfortable using the template structure :)

**Comment by jh-RLI** [comment]
I just noticed that they include a link to an external repository. You might want to update the link to the contributing.md of the open_MaStR repo. :)

File: .github/ISSUE_TEMPLATE/issue_template_bug.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Describe possible ideas for solution and evaluate advantages and disadvantages.
+
+## Context and Environment
+* Version used: 
+* Operating system: 
+* Environment setup and (python) version: 
+
+## Workflow checklist
+- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/rl-institut/super-repo/blob/develop/CONTRIBUTING.md)
```
Comment (chrwm) [inline .github/ISSUE_TEMPLATE/issue_template_bug.md]:
```suggestion
- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md)
```

File: .github/ISSUE_TEMPLATE/issue_template_feature.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+## Description of the issue
+
+Describe the problem in as much detail as possible. 
+
+## Ideas of solution
+
+Describe possible ideas for solution and evaluate advantages and disadvantages.
+
+## Workflow checklist
+- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/rl-institut/super-repo/blob/develop/CONTRIBUTING.md)
```
Comment (chrwm) [inline .github/ISSUE_TEMPLATE/issue_template_feature.md]:
```suggestion
- [ ] I am aware of the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md)
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+## Summary of the discussion
+
+Describe the findings of the discussion in the issue or meeting.
+
+## Type of change (CHANGELOG.md)
+
+### Added
+- Add a new class [(#)](https://github.com/rl-institut/super-repo/pull/)
```
Comment (chrwm) [inline .github/pull_request_template.md]:
```suggestion
- Add a new class [(#)](https://github.com/OpenEnergyPlatform/open-MaStR/pull/)
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+### Updated
+- Update a definition [(#)](https://github.com/rl-institut/super-repo/pull/)
```
Comment (chrwm) [inline .github/pull_request_template.md]:
```suggestion
- Update a definition [(#)](https://github.com/OpenEnergyPlatform/open-MaStR/pull/)
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+### Removed
+- Remove a broken link [(#)](https://github.com/rl-institut/super-repo/pull/)
```
Comment (chrwm) [inline .github/pull_request_template.md]:
```suggestion
- Remove a broken link [(#)](https://github.com/OpenEnergyPlatform/open-MaStR/pull/)
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+
+## Workflow checklist
+
+### Automation
+Closes #
+
+### PR-Assignee
+- [ ] 🐙 Follow the workflow in [CONTRIBUTING.md](https://github.com/rl-institut/super-repo/blob/develop/CONTRIBUTING.md)
```
Comment (chrwm) [inline .github/pull_request_template.md]:
```suggestion
- [ ] 🐙 Follow the workflow in [CONTRIBUTING.md](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md)
```

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] 📝 Update the [CHANGELOG.md](https://github.com/rl-institut/super-repo/blob/develop/CHANGELOG.md)
```
Comment (chrwm) [inline .github/pull_request_template.md]:
```suggestion
- [ ] 📝 Update the [CHANGELOG.md](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CHANGELOG.md)
```

File: .github/pull_request_template.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+- [ ] 📙 Update the documentation
+
+### Reviewer
+- [ ] 🐙 Follow the [Reviewer Guidelines](https://github.com/rl-institut/super-repo/blob/develop/CONTRIBUTING.md#40-let-someone-else-review-your-pr)
```
Comment (chrwm) [inline .github/pull_request_template.md]: @steull please add the "Reviewer Guidlines" to this repo & update the link or delete the line

**Review by chrwm — CHANGES_REQUESTED** [review]
Thanks for the initiative.
Please fix the requested changes and I'll approve. THX!

File: .github/pull_request_template.md (hunk 8 above)
Comment (chrwm) [inline .github/pull_request_template.md]:
```suggestion
- [ ] 🐙 Follow the [Reviewer Guidelines](https://github.com/OpenEnergyPlatform/open-MaStR/blob/production/CONTRIBUTING.md#40-let-someone-else-review-your-pr)
```

**Review by chrwm — APPROVED** [review]
@steull, I incorporated the changes.
Please merge today or tomorrow, before 11 a.m.. THX!

**Pull request merged**